collections/blueprints/pci.md

```diff
@@ -0,0 +1,268 @@
+touch card-form.js
+```
+
+In your editor, update your App component with the following code, replacing `PUBLIC_API_KEY` with the API Key from the previous step:
+
+```jsx
+import { 
+  useBasisTheory,
+  BasisTheoryProvider
+} from '@basis-theory/basis-theory-react';
+import { CardForm } from './card-form';
+
+export default function App() {
+  const { bt } = useBasisTheory('PUBLIC_API_KEY', { elements: true });
+
+  return (
+    <BasisTheoryProvider bt={bt}>
+      <CardForm />
+    </BasisTheoryProvider>
+  );
+}
+```
+
+Next, add the following code to your new Card Form component:
+
+```jsx
+import { 
+  useBasisTheory
+} from '@basis-theory/basis-theory-react';
+
+export const CardForm = () => {
+  const { bt } = useBasisTheory();
+
+  const submit = async () => {};
+
+  return (
+    <form onSubmit={submit} className="form">
+      <button type="button" onClick={submit} >
+        Submit
+      </button>
+    </form>
+  );
+}
+```
+
+This will initialize Basis Theory JS and add your new card form.
+
+### Add Card Element
+
+Now, we need to add a [`CardElement`](https://docs.basistheory.com/elements/#element-types-card-element) component to our form. This Element type offers a single line to capture the card number, expiration date, and CVC.
+
+Add the `CardElement` to our inputs:
+
+```jsx
+import { 
+  useBasisTheory,
+  CardElement
+} from '@basis-theory/basis-theory-react';
+```
+
+And inside of our `<form>` tag, add the following code:
+
+```jsx
+<CardElement id="card" />
+```
+
+Your card component should now look like this:
+
+```jsx
+import { 
+  useBasisTheory,
+  CardElement
+} from '@basis-theory/basis-theory-react';
+
+export const CardForm = () => {
+  const { bt } = useBasisTheory();
+
+  const submit = async () => {};
+
+  return (
+    <form onSubmit={submit} className="form">
+      <CardElement id="card" />
+
+      <button type="button" onClick={submit} >
+        Submit
+      </button>
+    </form>
+  );
+}
+```
+
+You can fully customize the [look and feel](./guides/secure/../../../guides/secure/style-elements-for-my-brand.md) to match your existing site and even listen to common [events](https://docs.basistheory.com/elements/#element-events) on all Elements components.
+
+## Store Cards
+
+Now that we have securely captured the card information, we need to securely store the card. 
+
+To do this, we will tokenize the data with Basis Theory. Basis Theory handles all of the secure encryption and storage of the cardholder data and returns a non-sensitive token identifier that can be stored in your database.
+
+### Tokenize the Card
+
+Inside of your `CardForm`, we will update the `submit` function to tokenize the card using the PCI-compliant [`card` Token Type](https://docs.basistheory.com/#token-types-card):
+
+```jsx
+const submit = async () => {
+  // Tokenize card with Basis Theory
+  const token = await bt.tokenize({
+    type: 'card',
+    data: bt.getElement('card'),
+  });
+
+  // Submit card token to API
```

Comment:
Nitpick, but I think we should clarify that this is our app's API, not the BT API

```suggestion
  // Submit card token to our Next.js app's API
```